BSB_lan_defs.h

```diff
@@ -7762,6 +7860,9 @@ PROGMEM_LATE const cmd_t cmdtbl2[]={
 {0x513D052E,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8560,  STR8560,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // [°C ] - Diagnose Erzeuger - Feststoffkesseltemperatur
 {0x513D04CF,  CAT_DIAG_ERZEUGER,    VT_PERCENT,       8568,  STR8568,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Drehzahl Festst'pumpe
 {0x513D0892,  CAT_DIAG_ERZEUGER,    VT_HOURS,         8570,  STR8570,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Betr'std Feststoffkessel
+{0x053D14A0,  CAT_DIAG_ERZEUGER,    VT_TEMP,          8585,  STR8585,  0,                    NULL,         FL_RONLY,     DEV_ALL}, // Regeltemperatur
+// todo: How to integrate this line? it's sent along with parameter 8585
+// {0x053D149F,  CAT_DIAG_ERZEUGER,    VT_TEMP,          ????,  STR8585_1, 0,                  NULL,         FL_RONLY,     DEV_ALL}, // Zusatzerzeugersollwert
```

Comment:
Not sure which parameter number to assign. The datagram is sent and displayed along with parameter 8585.

Reply:
Add the second line as 8586.